mycroft/skills/wolfram_alpha/__init__.py

```diff
@@ -184,16 +186,40 @@ def handle_fallback(self, message):
 
             self.speak(response)
         else:
-            self.speak("Sorry, I don't understand your request.")
+            if len(others) == 1:
+                self.speak("Sorry, I didn't understand " +
+                           utterance + ". Searching for " +
+                           others[0] + " instead")
+                self.handle_fallback(Message('intent_failure',
+                                             metadata={'utterance':
+                                                       others[0]}))
+            else:
+                self.speak("Sorry, I don't understand your request.")
```

Comment:
@ethanaward let's replace those old EN hard coded sentences dependency to .dialog and then provide more than one feedback.